binding-library/java/src/main/java/com/microsoft/azure/functions/signalr/SignalRMessage.java

```diff
@@ -16,6 +16,26 @@
  * @since 1.0.0
  */
 public class SignalRMessage {
+
+    /**
+     * Constructor
+     */
+    public SignalRMessage() {
+    }
+
+    /**
+     * Constructor
+     * 
+     * @param target    Target method to invoke on clients
+     * @param arguments Arguments to pass to target method
+     */
+    public SignalRMessage(String target, Object... arguments) {
```

Comment:
>public SignalRMessage( [](start = 4, length = 22)

Shall we also add one contractor with userId that for send to user and one contractor with groupname that for send to group?

Reply:
That should be handle in function. Ignore this.

---
In reply to: [261928906](https://github.com/Azure/azure-functions-signalrservice-extension/pull/53#discussion_r261928906) [](ancestors = 261928906)

Reply:
Been thinking about this one. Because they are both String, I'm not sure how we can differentiate. Potentially, we can remove userId and groupName from `SignalRMessage` and create subclasses `SignalRGroupMessage` and `SignalRUserMessage` that have those as fields and in the constructors. This is probably a better way to do it, but might be too late to make this big of a change.

Reply:
Yes. I'm thinking if it should be handled in app level or function level. It's not block issue. Let's consider it in next releasing cycle. 

---
In reply to: [261930864](https://github.com/Azure/azure-functions-signalrservice-extension/pull/53#discussion_r261930864) [](ancestors = 261930864)